install_config/registry/extended_registry_configuration.adoc

```diff
@@ -82,40 +82,40 @@ unnecessary downtime for pods and does not actually clear the cache.
 [[whitelisting-docker-registries]]
 == Whitelisting Docker Registries
 
-You can specify a whitelist of docker registries, allowing you to curate a set
-of images and templates that are available for download by {product-title}
-users. This curated set can be placed in one or more docker registries, and then
-added to the whitelist. When using a whitelist, only the specified registries
-are accessible within {product-title}, and all other registries are denied
-access by default.
+You can specify a whitelist of Docker registries, allowing you to curate a set 
+of images and templates that are available for download by {product-title} users.
+
+This curated set can be placed in one or more docker registries, and then added 
+to a whitelist defined in the *_/etc/containers/registries.conf_* file. 
+
+When using a whitelist, you can specify which secure and insecure registries 
+a user is allowed to download from, and which registries users cannot download
+from. 
 
 To configure a whitelist:
 
-. Edit the *_/etc/sysconfig/docker_* file to block all registries:
+. Edit the *_/etc/containers/registries.conf_* file to add or edit the following
+parameters as needed:
 +
 ----
-BLOCK_REGISTRY='--block-registry=all'
-----
-+
-You may need to uncomment the `*BLOCK_REGISTRY*` line.
-. In the same file, add registries to which you want to allow access:
-+
-----
-ADD_REGISTRY='--add-registry=<registry1> --add-registry=<registry2>'
-----
-+
-.Allowing Access to Registries
-----
-ADD_REGISTRY='--add-registry=registry.redhat.io'
-----
-+
-This example would restrict access to images available on the
-link:https://access.redhat.com/search/#/container-images[Red Hat Customer Portal].
+[registries.search] <1>
+registries = ["reg1.example.com", "reg2.example.com"]
+
+[registries.insecure] <2>
+registries = ["reg3.example.com"] 
 
-Once the whitelist is configured, if a user tries to pull from a container image registry
-that is not on the whitelist, they will receive an error message stating that
-this registry is not allowed.
+[registries.block] <3>
+registries = ['docker.io']
+----
+<1> Specify the secure registries that users can download images.
+<2> Specify the insecure registries that users can download images.  
```

Comment:
The `registries.insecure` list tells the client which registries are not secured with SSL/TLS, so that it knows not to try to set up a secure connection with them.  This list is independent of `registries.search`, though if there are insecure registries in the search list they should also be added here.